### ISSUE TITLE
change TTS voice

### ISSUE THREAD

**Issue by omsenjalia** [keithhb33/AI-Movie-Shorts#1]
is it possible to change TTS voice? if yes what options are available?

**Comment by TerraPlayz** [comment]
background audio too

**Comment by keithhb33** [comment]
Yes, it is possible to change the TTS voice, but you must implement a different TTS package yourself. The current one uses a free TTS system.

**Comment by keithhb33** [comment]
I will add you on discord. For the background music, I'm pretty sure it just randomly selects a song to use in the background music directory at the moment. This could easily be changed. When I have time this summer I hope to create a CLI version of the bot. Until then, it's current state will have to do. Feel free to change anything via a fork.

**Comment by omsenjalia** [comment]
> I will add you on discord. For the background music, I'm pretty sure it just randomly selects a song to use in the background music directory at the moment. This could easily be changed. When I have time this summer I hope to create a CLI version of the bot. Until then, it's current state will have to do. Feel free to change anything via a fork.

I am glad to hear back from you. my discord is `om.senjalia`. I did figure out the background music thing but I am still needing support for tts. I am using this bot to create Anime Recap Videos. But they are too small (2 or 3 minutes).
